bip39-create-accounts/.env.example

```diff
@@ -0,0 +1,3 @@
+OPERATOR_ID="YOUR_OPERATOR_ID"
+OPERATOR_KEY="YOUR_OPERATOR_ED25519_PRIVATE_KEY_IN_DER_FORMAT"
+SEED_PHRASE="YOUR_BIP39_SEEDPHRASE"
```

Comment:
Consider providing a sample bip39 seed phrase (with the appropriate disclaimer) to enable devs to run this quickly without having to step out of this tutorial to generate a new seed phrase

Reply:
Will do in follow up PR!